addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -17,13 +17,13 @@
                             <i t-elif="!user.im_status" class="fa fa-fw fa-question-circle" title="No IM status available"/>
                         </span>
                     </span>
-                    <div class="d-flex flex-column o_card_user_infos">
-                        <span class="fw-bold" t-esc="user.name"/>
+                    <div class="d-flex flex-column o_card_user_infos overflow-hidden">
+                        <span class="fw-bold text-truncate" t-esc="user.name" t-att-title="user.name"/>
```

Comment:
`I would replace `text-truncate` by a `text-wrap`, and remove `t-att-title` as it becomes useless.

I personally don't like `text-truncate`, I think this is overused and enforces style consistency at the cost of worse UX experience: text can sometimes not be visible, and reliance on `title` sucks. The `title` is even shown when the text doesn't overflow, so it sucks to have so many hover area polluting UI with floating stuffs.

So when we can, I prefer showing the explicit text in full and manage overflow nicer e.g. here with `text-wrap`.
This technique doesn't work all the time, I get `text-truncate` is handy especially for inline elements. But for block elements like the name, there's no reason to dismiss `text-wrap`.

Reply:
Well that's actually [the discussion we had here](https://github.com/odoo/odoo/pull/165172#discussion_r1599727346) in which I mentioned that we should allow the title to wrap rather than using truncation so we should not enforce a class on the username, indeed.

Reply:
Indeed I didn't saw the very-end of the message where you suggested to wrap the name. In the discussion I was more concerned by long names that was not handled, I suggested `text-truncate` just as a mean to manage long text. But I should have taken more time to suggest `text-wrap` immediately as to not waste time with this 2nd review. Sorry!

Reply:
Did we not do wrapping at the end?

Reply:
Not needed as the popover already comes with `text-wrap: wrap`;